test/integration/manifests/cns/daemonset-windows.yaml

```diff
@@ -104,13 +104,13 @@ spec:
                   apiVersion: v1
                   fieldPath: spec.nodeName
       initContainers:
-        - name: init-cni-dropgz
-          image: acnpublic.azurecr.io/cni-dropgz:CNI_DROPGZ_VERSION
+        - name: cni-installer
+          image: acnpublic.azurecr.io/cni-dropgz:latest
           imagePullPolicy: Always
-          command: ["%CONTAINER_SANDBOX_MOUNT_POINT%/dropgz.exe"]
+          command: ["%CONTAINER_SANDBOX_MOUNT_POINT%/dropgz"]
```

Comment:
Should we keep .exe? Since this is windows?

Reply:
don't need it

Reply:
So it's still able to invoke just `dropgz`, even though we call it `dropgz.exe` here?
https://github.com/Azure/azure-container-networking/blob/2a90a322b1d22708013cd12a3213fd1d917edeb3/dropgz/build/windows.Dockerfile#L37

Reply:
no, this is the old image and there are no changes to how it is used. the reference in the ds is to the new image